pyproject.toml

```diff
@@ -147,6 +147,9 @@ lint.per-file-ignores."project_euler/problem_099/sol1.py" = [
 lint.per-file-ignores."sorts/external_sort.py" = [
   "SIM115",
 ]
+
+lint.flake8-builtins.builtins-allowed-modules = [ "base64", "wave" ]
```

Comment:
Let’s rename the files instead like base64_cypher.py and wave_string.py

Reply:
Done